src/main/java/flight/reservation/flight/ScheduledFlight.java

```diff
@@ -62,9 +73,13 @@ public int getCapacity() throws NoSuchFieldException {
         }
         throw new NoSuchFieldException("this aircraft has no information about its capacity");
     }
+*/
 
+    public int getCapacity() {
```

Comment:
Much cleaner, let plane decide about its capacity. Good! However, cast can be avoided by changing type of aircraft member.